Sources/SwiftProtobuf/AnyMessageStorage.swift

```diff
@@ -202,7 +205,18 @@ internal class AnyMessageStorage {
       // never inserted.
       break
 
-    case .contentJSON:
+    case .contentJSON(let contentJSON, let options):
+      do {
+        guard let messageType = Google_Protobuf_Any.messageType(forTypeURL: _typeURL) else {
```

Comment:
Shouldn't this go above the `do {...` instead of inside it?  i.e. - why catch and rethrow it?

Reply:
Done.